apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/web.tsx

```diff
@@ -24,15 +24,15 @@ const categories: { Other?: ICategory; [name: string]: ICategory } = {
     TextField: {},
     Toggle: {}
   },
-  'Galleries and Pickers': {
+  'Galleries & Pickers': {
```

Comment:
Dumb question--have you confirmed that this doesn't need to be escaped?

Reply:
React should automatically handle escaping in rendered string literals.